apps/web/src/pages/studies/index.tsx

```diff
@@ -212,6 +220,13 @@ export const getServerSideProps = withServerSideProps(Roles.SponsorContact, asyn
     })
 
     const filters = getFiltersFromQuery(context.query)
+    let entry: Entry<TypeBannerSkeleton> | null
+
+    try {
```

Comment:
We've got multiple try/catch statements here? What behaviour are we actually trying to achieve?

Inside the `getNotificationBanner()` method you're already catching any unhandled exceptions that occur from the Contentful call and returning 'null' if anything goes wrong, so this try/catch is essentially doing nothing.

Is entry being null a valid scenario, or should we in fact be blowing up if the Contentful call fails?

Reply:
The catch inside `getNotificationBanner()` is valid as contentful will throw an error when no entry exists so we are handling this and returning null to not show the banner which meets the client requirements of only showing the component when it's published

I did consider removing this try/catch too but it allows the application to continue to run if the assertion fails for some reason outside of the CI/CD, happy to remove and allow it to bubble up

Reply:
If CONTENTFUL_BANNER_ENTRY_ID should always have a value, we should let the assert() throw the relevant error.

if CONTENTFUL_BANNER_ENTRY_ID can be null, we should just handle this inside the getNotificationBanner() method and return a null entry, rather than have two sets of try/catches. Exceptions are expensive.